packages/legend-application-studio/src/stores/StoreRelational_LegendStudioApplicationPlugin_Extension.ts

```diff
@@ -97,4 +120,38 @@ export interface StoreRelational_LegendStudioApplicationPlugin_Extension
    * Get the list of renderers for the editor for a relational database authentication strategy.
    */
   getExtraAuthenticationStrategyEditorRenderers?(): AuthenticationStrategyEditorRenderer[];
+
+  // --------------------- relational database connection post-processor ------------------
+
+  /**
+   * Get the list of the supported type for post-processors.
+   *
+   */
+  getExtraPostProcessorClassifiers?(): string[];
+
+  /**
+   * Get the list of classifiers for a post-processor.
+   */
+  getExtraPostProcessorClassifierGetters?(): PostProcessorTypeGetter[];
+
+  /**
+   * Get the list of creators for post-processor given the type specifier.
+   */
+  getExtraPostProcessorCreators?(
+    connectionValueState: RelationalDatabaseConnectionValueState,
```

Comment:
Don't think we need parameters for these. how do you justify this? Same question for `getExtraPostProcessorStateCreators()`, else, can we put these param in `PostProcessorCreator` instead?

Reply:
The `getExtraPostProcessorStateCreators` params are used since the method itself depending on type will call a method (relationalDatabaseConnection_addSpecialPostProcessor) example or (relationalDatabaseConnection_addSpecialMapperPostProcessor), and so those methods require connectionValueState I believe. But I will try plopping into `PostProcessorCreator` 👍

Reply:
hm, in general though, it's super super rare that you need the params here for the `getExtra...` after all it's a getter for the list of extensions, usually no logic is done till you call the extension method itself.

Reply:
I am probably doing something wrong in legendary I will check it there 🤔